package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "test": "spago run --main Test.Main",
     "dev": "BROWSER_RUNTIME=1 webpack-dev-server --progress --hot",
-    "build": "BROWSER_RUNTIME=1 webpack --mode=production"
+    "build": "BROWSER_RUNTIME=1 webpack --mode=production",
+    "bundle-seabug": "BROWSER_RUNTIME=1 webpack --config ./webpack.config.seabug-bundle.js"
```

Comment:
Do we want `--mode=production` for this as well?

Reply:
it is set in the config, i can move it here if that's better

Reply:
Ah OK, sorry I missed that